guilib/measure/dialogs/constants.py

```diff
@@ -0,0 +1,15 @@
+"""Module constants of viperleed.guilib.measure.dialogs.
```

Comment:
Consider if it makes sense to move this module "one level up", to guilib/dialogs